nzedb/processing/tv/TV.php

```diff
@@ -567,247 +460,230 @@ public function countEpsByVideoID($videoId)
 	}
 
 	/**
-	 * Supplementary to parseNameEpSeason
-	 * Cleans a derived local 'showname' for better matching probability
-	 * Returns the cleaned string
+	 * Parses a release searchname for specific TV show data
+	 * Returns an array of show data
 	 *
-	 * @param $str
+	 * @param $relname
 	 *
-	 * @return string
+	 * @return array|bool
 	 */
-	public function cleanName($str)
+	public function parseShowInfo($relname)
```

Comment:
public function parseInfo($relname)
No need for 'Show' in everything, it's the whole point of these classes, no need to repeat it except where it is specifically being used. As a side note, 'show' is a culturally specific term not universally used.